File: soalNo06.js
// Buatlah program yang menerima input sebuah bilangan bulat `n`,
// dan menampilkan deret kelipatan 3 sebanyak `n` bilangan.

// jalankan dengan node js

// script input node js
const rl = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Masukan bilang bulat : ", (angka) => {
  const n = parseInt(angka); // membuat inputan menjadi Integer
  const m = 3; // angka kelipatan bisa di rubah
  deretKelipatan(n, m);
  rl.close();
});

// code disini
// fungsi menampilkan deret kelipatan tiga
function deretKelipatan(n, m) {
  let deretKelipatan = 0;
  for (let i = m; i <= n; i += m) {
    deretKelipatan = i;
    console.log(deretKelipatan);
  }
}

File: soalNo06.py
angka = input("Masukan bilang bulat: ")
n = int(angka)
m = 3

def deret_kelipatan(n, m):
    deret_kelipatan = 0
    for i in range(m, n + 1, m):
        deret_kelipatan = i
        print(deret_kelipatan)

deret_kelipatan(n, m)
